pyproject.toml

```diff
@@ -18,3 +18,42 @@ typeCheckingMode = "strict"
 
 [tool.isort]
 profile = "black"
+
+[tool.ruff]
+select = ["E", "W", "I"]
+ignore = ["E741"]
+
+# Allow autofix for all enabled rules (when `--fix`) is provided.
```

Comment:
the comment says `ALL` but there's an explicit list? does this list correspond to the `ALL` option or is there a change? if so, why and which?

Reply:
Ok, let's remove default config.